packages/create-app.template.expo.v2/templates/foundry/client.ts.hbs

```diff
@@ -0,0 +1,13 @@
+import { Client, createClient, createPlatformClient, PlatformClient } from '@osdk/client';
+import { FOUNDRY_URL } from './osdkConst';
+import { getValidAuthToken } from './Auth';
+import { $ontologyRid } from "{{osdkPackage}}";
+
+
+export const client: Client = createClient(
+    FOUNDRY_URL, 
+    $ontologyRid, 
+    getValidAuthToken,
+);
+
+export const platformClient: PlatformClient = createPlatformClient(FOUNDRY_URL, getValidAuthToken);
```

Comment:
I guess we can but as a template I thought of showing both options. prefer to keep it as is but no strong opinion.

Reply:
Ok I also don't have strong opinions, only asked because I think the other templates only give you one client so was thinking we would want to keep parity, but I'll leave that up to you!